internal/v2/datasources/apono_managed_groups_datasource_unit_test.go

```diff
@@ -49,32 +49,20 @@ func TestAponoManagedGroupsDataSource(t *testing.T) {
 		mockListResponse := &client.PublicApiListResponseGroupPublicV1Model{
 			Items: []client.GroupV1{
 				{
-					ID:   "g-123456",
-					Name: "test-group-1",
-					SourceIntegrationID: client.OptNilString{
-						Value: "source-int-1",
-						Set:   true,
-					},
-					SourceIntegrationName: client.OptNilString{
-						Value: "Source Integration 1",
-						Set:   true,
-					},
+					ID:                    "g-123456",
+					Name:                  "test-group-1",
+					SourceIntegrationID:   client.NewOptNilString("source-int-1"),
```

Comment:
as you cam see... I'm using ogen helper functions which make it a little bit cleaner.
this will be all over the PR.